Comments not in UTF-8 - error with newer versions of LLVM

<!---
Thank you for reporting a problem and suggesting improvements. Please provide the below information to make sure we can effectively deal with the issue reported. For the most precise version information, see the first line of console output or run with --version. Please attach or include example code that allows us to reproduce the problem.
--->

CBMC version: 5.75.0
Operating system: MacOS 10.15.7
Exact command line resulting in the issue: cmake --build build -- -j 8
What behaviour did you expect: Successful build
What happened instead: clang error: cbmc-git/src/big-int/bigint-func.cc:5:9: error: invalid UTF-8 in comment [-Werror,-Winvalid-utf8]

The offending character is a ü.

I was having trouble building with Homebrew (which might be on me for using an unsupported version of MacOS) so I followed the compiling instructions included in the repo.

It looks like clang has recently added a new warning for non-UTF-8 comments:
https://reviews.llvm.org/D128059

I'm guessing that the encoding used in the source files is Windows-1252 -- I haven't tried converting the source files yet -- but since clang has added this compiler warning (or in my case an error due to the -Werror flag), I figure there either needs to be a change in CMakeLists.txt or in the source file encoding so that folks can compile without issue.

--- a/src/big-int/bigint-func.cc
+++ b/src/big-int/bigint-func.cc
@@ -2,7 +2,7 @@
 // Author: Dirk Zoller
 // References:
 //
-// [1] Hüttenhofer/Lesch/Peyerimhoff, Mathematik in Anwendung mit C++
+// [1] HÃ¼ttenhofer/Lesch/Peyerimhoff, Mathematik in Anwendung mit C++
 //     Heidelberg, Wiesbaden 1994
 
 #include "bigint.hh"

